rascal-vscode-extension/src/auto-jvm/JavaLookup.ts

```diff
@@ -173,7 +179,17 @@ async function downloadJDK(): Promise<string> {
     }
     const choice = await vscode.window.showInformationMessage("Select which OpenJDK provider you prefer", {modal:true}, ...options);
 
-    const result = await vscode.window.withProgress({
+    if(!choice){
+        throw new Error("User setup required");
+    }
+
+    const result = await downloadJDKWithProgress(choice);
+    vscode.window.showInformationMessage(`Finished downloading ${choice} JDK, rascal will now start`);
```

Comment:
pls fix the sentence while we are at it: `Finished downloading ${choice} JDK. Rascal will now start.`